.action_templates/steps/setup-kind-cluster.yaml

```diff
@@ -3,6 +3,9 @@
     curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.11.1/kind-linux-amd64
     chmod +x ./kind
     ./kind create cluster
+- name: Create Directories
+  run: |
+    docker exec kind-control-plane  mkdir -p /opt/data/mongo-data-0 /opt/data/mongo-data-1 /opt/data/mongo-data-2 /opt/data/mongo-logs-0 /opt/data/mongo-logs-1 /opt/data/mongo-logs-2
```

Comment:
The only way we can test this on Kind is to exec into the container and create six volumes (3 pods with two volumes each for logs and data).

Note: This will be done for each test, even the one that do not need it, but I don't think it is worth separating the logic